cargo-sqlx/README.md

```diff
@@ -0,0 +1,14 @@
+# cargo-sqlx
+
+Sqlx migrator runs all `*.sql` files under `migrations` folder and remembers which ones has been run. 
+
+Database url is supplied through either env variable or `.env` file containing `DATABASE_URL="postgres://postgres:postgres@localhost/realworld"`.
+
+##### Commands
+- `add <name>` - add new migration to your migrations folder named `<timestamp>_<name>.sql` 
+- `run` - Runs all migrations in your migrations folder
+
+
+##### Limitations
+- No down migrations! If you need down migrations, there are other more feature complete migrators to use.
```

Comment:
I don't think down migrations should exist. 

 * https://laraveldaily.com/still-need-migrations-taylor-says-no/

Generally I would recommend leap-frogging migrations as a migration should never be _breaking_ on its own as that would equal downtime and make automatic migrations mostly pointless anyway. 

If you made a mistake in a schema migration, make another to roll it back.

Reply:
I agree. The only place where I use down migrations is during development. But that can be worked around if you have a good seed. The refresh feature mentioned in the podcast could be useful though.

Reply:
I'll open an issue to talk about the commands but we should definitely have a refresh, recycle, or otherwise command that drops, creates, and then runs all migrations. Infinitely useful in development.

Reply:
> I'll open an issue to talk about the commands but we should definitely have a refresh, recycle, or otherwise command that drops, creates, and then runs all migrations. Infinitely useful in development.

I broadly agree but want to add a caveat here is around seed data.

> But that can be worked around if you have a good seed

While this is an important point, good seed data typically targets the latest migration. If you specifically want to test and refine the migration against known heterogenous data (e.g. adding a column derived from data in an existing column), this becomes quite clumsy. Down migrations--_even if_ they were only an option in dev somehow--are really a nice option for this kind of thing. In my experience working with larger and/or older datasets (1TB+ in my most recent case, larger before that) these kinds of migrations are common and laborious and having to blow away your entire DB as the only option for testing a migration _in the presence of_  such data would be a total PITA.